kaggle_environments/envs/werewolf/visualizer/default/src/world/Environment/SkySystem.js

```diff
@@ -89,7 +89,7 @@ export class SkySystem {
 
   createMoonMesh() {
     const textureLoader = new this.THREE.TextureLoader();
-    const moonTexture = textureLoader.load('/static/moon_texture.jpg');
+    const moonTexture = textureLoader.load(`${import.meta.env.BASE_URL}static/moon_texture.jpg`);
```

Comment:
all these imort.meta.env.BASE_URL changes are needed to make sure the prod iframe fetches the assets from the correct subfolder, as otherwise it will try to fetch them from the top-level kaggleusercontent.com folder.